cmd/fleet/serve.go

```diff
@@ -423,9 +423,12 @@ the way that the Fleet server works.
 				mdmPushService = nanomdm_pushsvc.New(mdmStorage, mdmStorage, pushProviderFactory, nanoMDMLogger)
 			}
 
+			cronSchedulesService := &fleet.CronSchedules{Schedules: make(map[string]fleet.CronSchedule)}
```

Comment:
Wonder if we should wrap this in a `NewCronSchedules` constructor.

Reply:
I was thinking the same thing :)